core/src/CorePlatformInformationBase.cs

```diff
@@ -0,0 +1,72 @@
+﻿//----------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Core
+{
+    internal abstract class CorePlatformInformationBase
```

Comment:
>Core [](start = 28, length = 4)

IMO: the namespace already says core and thus the class does not need that. Unless each platform is also having a base.

Reply:
we already have a platforminformationbase in adal and msal. I named it so, to drive clarity. I can also rename adal/msal class to AdalPlatformInformationBase. what do you think?

---
In reply to: [161031253](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/941#discussion_r161031253) [](ancestors = 161031253)

Reply:
Got it. imo keep it as is. 

---
In reply to: [161134280](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/941#discussion_r161134280) [](ancestors = 161134280,161031253)